doc/user-manual/language/universe-levels.lagda.rst

```diff
@@ -84,6 +84,54 @@ With this definition, we have, for example:
     Nat x Set : Set₁
     Set × Set : Set₁
 
+Intrinsic level properties
+--------------------------
+
+.. _intrinsic-level-properties:
+
+Levels and their associated operations have some properties
+which are internally and automatically solved by the compiler.
+This means that we can replace some expressions with others,
+without worrying about the expressions
+for their corresponding levels matching exactly.
+
+For example, for some `F : (l : Level) -> Set l`, `l1 l2 : Level`,
```

Comment:
Such examples should be checked by Agda in order to avoid mistakes and rotting documentation.  Please use `postulate` and rework your example so that it is checked.  The command to run the checks on the user-manual is
```
make user-manual-test
```
run from the project root.

Reply:
I've fixed the "cosmetic" changes, but this doesn't "just work" (doesn't find some tests directory). I assume that I additionally have to do something like `make && make test` first (so I'm waiting on that right now). Should those perhaps be added to the `user-manual-test` target in the makefile?

Reply:
I also don't have a `fix-whitespace` executable (and some of my make targets seemed to fail because of this) - I figure I'm supposed to get this from the [corresponding uploaded executable](https://hackage.haskell.org/package/fix-whitespace) on Hackage, but I couldn't find any documentation that I'm supposed to do this (e.g. in `HACKING.md`) - should I(/we) make another pr mentioning that somewhere? Or am I perhaps "just doing it wrong"?